minpy/dispatch/policy.py

```diff
@@ -70,4 +86,13 @@ def fst(t):
         x[1], array.Value) and x[1].marked_for_bp, kwargs.items())))
     available = reg.iter_available_types(name, bp_args, bp_kwargs)
     preference = plc.decide(available, args, kwargs)
+    if preference is None:
```

Comment:
That's great job in adding these exceptions. Could you add some comments in the base Policy.decide function to mention that if no candidate is suitable, just return `None`?